src/commands/mod.rs

```diff
@@ -726,6 +729,30 @@ struct RestoreArgs {
     revision: Option<RevisionArg>,
 }
 
+
+/// Run a command across a set of revisions.
+///
+///
+/// All recorded state will be persisted in the `.jj` directory, so occasionally
+/// a `jj run --clean` is needed to cleanup disk space.
```

Comment:
nit: should be "clean up", not "cleanup"

Reply:
Done.